src/components/pages/block/block-details.tsx

```diff
@@ -33,14 +34,21 @@ const BlockDetails = ({ block }: { block: BlockWithTransactions }) => (
     </DescriptionListItem>
     <BlockExtraData extraData={block.extraData} />
     <Separator />
-    <DescriptionListItem title="Hash">{block.hash}</DescriptionListItem>
+    <DescriptionListItem title="Hash">
+      {block.hash}
+      <CopyButton content={"Copy Hash to clipboard"} copy={block.hash} />
```

Comment:
Remove curly braces when specifying string props, should be `content="Copy Hash to clipboard"`